mgmt-fixes/deploy/helm/mgmt-fixes/Chart.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: Fixes
+description: Any fixes that need to be uplied but have not (yet) been upstreamed
```

```suggestion
description: Any fixes that need to be applied but have not (yet) been upstreamed
```
I think this was the typo